### PR TITLE
small adjustments

### PR DESCRIPTION
**Pull request by adRn-s**

### CODE REVIEW

File: src/npr/ont_pipeline.py [hunk 1]
```diff
@@ -69,20 +69,22 @@ def filter_flowcell(json, config):
...
-    fc_dir = os.path.basename(fc_dir.rstrip('/'))  
-    if fc_dir in config['ignore']['dirs']:
-        print("ignore fc_dir {} because of config ".format(fc_dir))
-        return True
-    if fc_id in config['ignore']['flowcells']:
-        print("ignore fc_id {} because of config ".format(fc_id))
-        return True
-
-    return False
+    ## This is dead code, given ^ -- Adrian
```
Comment (caballero) [inline src/npr/ont_pipeline.py:72]: the it can be removed

File: src/npr/rules/1_basecalling.smk
Comment (caballero) [inline src/npr/rules/1_basecalling.smk:1]: this is the guppy pipeline all rules/* can be removed

File: src/npr/rules/3_qc.smk
Comment (caballero) [inline src/npr/rules/3_qc.smk:1]: this is the guppy pipeline all rules/* can be removed

File: src/npr/rules/4_data_transfer.smk
Comment (caballero) [inline src/npr/rules/4_data_transfer.smk:1]: this is the guppy pipeline all rules/* can be removed

File: src/npr/rules_dorado/07_modbed.smk [hunk 2]
```diff
@@ -29,7 +29,7 @@ rule bam2modbed:
     conda:
         "ont-ppp-modkit"
     shell:'''
-        if [[ "{params.do_modbed}" == "do_modbed" ]]; then
+        if [[ "{params.do_modbed,,}" == "yes" ]]; then
```
Comment (caballero) [inline src/npr/rules_dorado/07_modbed.smk:32]: extra commas?

**Review by caballero — APPROVED** [review]
consider to remove all src/npr/rules as those are guppy specific, for the current pipeline src/npr/rules_dorado are the rules applied

File: src/npr/rules_dorado/07_modbed.smk (hunk 2 above)
Comment (adRn-s) [reply src/npr/rules_dorado/07_modbed.smk:32]: TIL they convert the content of the bash variable to lowercase (🗡️ it remains to be seen if it works in all the bashes out there, like other OSes), but it did work on CentOS 7 👼🏽

**Pull request merged**